packages/design-system/.storybook/docs/Badge.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styled from 'styled-components';
+import tokens from '@talend/design-tokens';
+
+import { Status } from './StatusTable';
+
+const A = styled.a.attrs(({ href }) => ({
```

Comment:
What are the benefits of keeping Styled Components as a devDep for the docs components?

Reply:
None, I will fix

Reply:
What a man